tests/libcxx/run_tests_locally.sh

```diff
@@ -16,29 +16,18 @@ FILE=$1
 while read test; do
   echo "$test"
   OUTPUT=$(2>&1 timeout 10 make one TEST=$test )
+  RETURN_VAL=$?
   echo $OUTPUT >> temp_$FILE.output
-  HAS_UNHANDLED_SYSCALL=$(2>&1 timeout 10 make one TEST=$test | grep "unhandled")
-  if [ -z "$HAS_UNHANDLED_SYSCALL" ]
+  TERMINATED=$(echo "$OUTPUT" | grep -F 'terminate called') # check for termination due to exceptions
+  if [[ "$RETURN_VAL" -eq "0" && -z $TERMINATED ]]
   then
-    # No unhandled syscall
-    FAILED=$(echo "$OUTPUT" | grep failed)
-    FAIL_ENCLAVE=$(echo "$OUTPUT" | grep OE_ENCLAVE_ABORTING)
-    TIMED_OUT=$(echo "$OUTPUT" | grep -F '[one] Terminated')
-    TERMINATED=$(echo "$OUTPUT" | grep -F 'terminate called')
-    if [[ -z $FAILED && -z $FAIL_ENCLAVE && -z $TIMED_OUT && -z $TERMINATED ]]  
-    then
-      echo $test >> temp_passed.output
-    else
-      echo $test >> temp_other_errors.output
-    fi
+    echo $test >> temp_passed.output
   else
-    echo "$test: $HAS_UNHANDLED_SYSCALL" >> temp_unhandled_syscalls.output
+    echo $test >> temp_other_errors.output
```

Comment:
```suggestion
    echo $test >> temp_failed.output
```

Reply:
I named this other_errors since we put half of them in unhandled syscalls, but I have thought about making this change before - since we dont have any unhandled syscall errors in libcxx. Will add.